.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +23,4 @@ While we look over every pull request, we maintain a focus on this project's cur
 
 - [ ] I have added tests.
 - [ ] I have added necessary documentation.
+- [ ] I have auto formatted using clang-format or yapf.
```

Comment:
clang-format/yapf might sound better lol idk

Reply:
Gonna leave this for now and we can iron out the exact language when we also add the clang-tidy